definitions/private-law/json/AuthorisationCaseField/SummaryTab/AuthorisationCaseField.json

```diff
@@ -256,7 +256,7 @@
     "CaseTypeID": "PRLAPPS",
     "CaseFieldID": "confidentialDetails",
     "UserRole": "caseworker-privatelaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRUD"
```

Comment:
same as above comment

Reply:
this is not needed here for summary tab so replacing it with "R"